Example/Auth/Tests/FIRAuthTests.m

```diff
@@ -2283,8 +2270,9 @@ - (void)mockSecureTokenResponseWithError:(nullable NSError *)error {
  */
 - (void)enableAutoTokenRefresh {
   XCTestExpectation *expectation = [self expectationWithDescription:@"autoTokenRefreshcallback"];
-  [[FIRAuth auth].app getTokenForcingRefresh:NO withCallback:^(NSString *_Nullable token,
-                                                                NSError *_Nullable error) {
+  id<FIRAuthInterop> auth = FIR_COMPONENT(FIRAuthInterop, [FIRAuth auth].app.container);
```

Comment:
I think it makes sense to directly use Auth's call here instead of going through the container, and having a separate test to verify that the component is registered properly and performs as expected (essentially this test, but using the test component container if possible.

Reply:
Oh yes, this whole roundabout way of getting the instance is silly. I just needed to cast it to the protocol.